scenarioo-client/test/e2e/pages/stepPage.ts

```diff
@@ -238,18 +238,22 @@ class StepPage {
         return expect(element(by.id('comparison-tab')).isPresent()).toBeFalsy();
     }
 
-    async assertStepComparisonSideBySideViewIsActive() {
+    async assertStepComparisonSideBySideViewIsActive(otherScreenVisible: boolean) {
```

Comment:
Imho this should be a separate method/function not a flag. The usage of the method is strange to me.

Reply:
Added separate function.